services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_core/_scheduler.py

```diff
@@ -287,7 +287,7 @@ async def _await_task(task: asyncio.Task) -> None:
 
     async def is_service_awaiting_manual_intervention(self, node_uuid: NodeID) -> bool:
         """returns True if services is waiting for manual intervention"""
-        return _scheduler_utils.service_awaits_manual_interventions(
+        return await _scheduler_utils.service_awaits_manual_interventions(
```

Comment:
Time to start taking action to avoid this unfortunate errors 

https://github.com/ITISFoundation/osparc-simcore/issues/4071